lib/shipit.rb

```diff
@@ -291,7 +291,7 @@ def revision_file
   end
 
   def secrets
-    Rails.application.secrets
+    Rails.application.credentials
```

Comment:
Note that all `shipit-engine` users will have to migrate from secrets to credentials, so this is likely changelog worthy.